static/styles/landing-page.scss

```diff
@@ -2480,7 +2480,7 @@ nav ul li.active::after {
 .portico-landing.integrations .portico-page-subheading {
     font-weight: 400;
     font-size: 1.1em;
-    color: #aaa;
+    color: hsl(0, 0%, 67%);
```

Comment:
Previously in one of the commits this was changed to `hsl(0, 0%, 66%)`
There are a number of such values in the code base.

https://github.com/zulip/zulip/commit/d1d1906460f4e8455ecdb189ffe8c7475e373668#diff-d865d77a269f54682ba4b6811a798d2cR88